### DISCUSSION TITLE
Rifiuti does not work on XP

### DISCUSSION THREAD

**Discussion by pedro22** [abelcheung/rifiuti2#45] [General]
Rifiuti does not work on XP 32bit - I type "rifiuti INFO2" and get: 'INFO2' does not exist. I try "rifiuti -l INFO2" and get: "Must specify exactly one INFO2 file as argument.", but rifiuti -? correctly displays. 

My system is on disk I: not C: - Can this be a problem?

**Comment by abelcheung** [comment]
If you don't provide the exact path of INFO2 file, the program expects it to be found inside current directory. Usually INFO2 files can be found in `\RECYCLER\S-1-......\` directory on each drive (actual path varies from system to system). And note that this file is hidden by default.

**Comment by pedro22** [comment]
I type: rifiuti -l "U:\RECYCLER\S-1-5-21-839522115-1637723038-725345543-1003\INFO2" and get: Must specify exactly one INFO2 file as argument.

what am I doing wrong?

**Comment by abelcheung** [comment]
@pedro22 Sorry for postponing the reply for so long. In your case, the command line argument was wrong. You can first try running without the `-l` option; if it dumps error saying you *must* use `-l` option, then extra argument after `-l` option is needed, specifying the codepage used in the copy of Windows which generates your `INFO2` file.

For example, if the `INFO2` file originates from English version of Windows, then use
`rifiuti -l CP1252 <your_info2_file>`

**Comment by abelcheung** [comment]
Closing issue for now, feel free to reopen if same problem in usage is still not resolved.
